doc/release-notes-bip35.md

```diff
@@ -0,0 +1,14 @@
+Notable changes
+===============
+
+P2P and network changes
+-----------------------
+* Deprecate the BIP35 `mempool` message due to disuse and poor privacy.
```

Comment:
"Deprecate" means "we still support this, but we recommend you don't use it". cf https://en.wikipedia.org/wiki/Deprecation